src/services/pathContents/pull/dir/constants.ts

```diff
@@ -0,0 +1,148 @@
+import { z } from 'zod'
+
+import { MissingHeadReportSchema } from 'services/comparison'
+import { UnknownFlagsSchema } from 'services/impactedFiles/schemas'
+
+const MissingCoverageSchema = z.object({
+  __typename: z.literal('MissingCoverage'),
+  message: z.string(),
+})
+
+const UnknownPathSchema = z.object({
+  __typename: z.literal('UnknownPath'),
+  message: z.string(),
+})
+
+const RepositoryConfigSchema = z.object({
+  indicationRange: z
+    .object({
+      upperRange: z.number(),
+      lowerRange: z.number(),
+    })
+    .nullable(),
+})
+
+const PathContentFileSchema = z.object({
+  __typename: z.literal('PathContentFile'),
+  name: z.string(),
+  path: z.string().nullable(),
+  hits: z.number(),
+  misses: z.number(),
+  partials: z.number(),
+  lines: z.number(),
+  percentCovered: z.number(),
```

Comment:
think all of these except name are nullable, unless you've verified they're not.

Reply:
They are not nullable. Double checked API. Will fallback to 0.